README.md

```diff
@@ -1217,6 +1235,73 @@ candidates. But even so, sophisticated negotiators can often get better offers.
+facilitate the negotiation, then follow it with a **written offer** if it seems like you’re
+close to a point where you’ll agree.
+The written offer takes the form of an **offer letter**, which is just the summary sent to
+you, typically with an expiration date and other details and
+[paperwork](#documents-and-agreements). If you are ready to accept the terms of the offer
+letter, you can sign the offer letter.
+
+Although companies often want you to sign right away, to save time and effort, if you’re
+doing it thoughtfully, you’ll also be talking to the company (typically with a hiring
+manager, your future manager, or a recruiter, or some combination) multiple times before
+signing. This helps you negotiate details and also just get to know the people, the
+company, and the role, so you make a good decision.
+
+Things to look for in the offer letter include:
+
+- Title and level:
+  What your role is officially called, who you report to, and what level of seniority your
+  role is within the company.
+- Salary: What you’re paid.
+- Equity compensation:
+  You know what this is now.
+- Bonus: Additional cash you’ll get on a regular basis, if the company has a plan for this.
+- Signing bonus:
+  Cash you get just for signing.
+  Note signing bonuses usually have some strings attached—for example you will have to pay
+  it back if you leave the company within 12 or 24 months.
+
+While the details may not be included in your offer letter, you’ll also want to discuss
+and get full information on:
+
+- Benefits: All the other perks from the job, including health insurance and retirement and
+  free food.
+- Accommodations:
+  Other aspects of the job that might matter to you, like time off, ability to work from
+  home, flexible hours, and so on.
+
+Some general notes on these components, including
+[several by Cristina Cordova](https://twitter.com/cjc/status/984094472190349312):
+
+- Early stage startups will focus on salary and equity and (if they are funded) benefits.
+  Bonus and signing bonus are more common in larger cash-flush companies.
+- Bonus is usually standardized to the company and your level, so not likely to be something
+  you can negotiate.
+- Signing bonus is highly negotiable.
+  This doesn’t mean any company will give large signing bonuses, but it’s often more
+  feasible, since it varies candidate by candidate, and unlike salary and bonus, it’s a
+  one-time cost to the company.
+  Depending on terms you may need to pay it back if you leave the company right away.
+
+### Startup offers
+
+Because startups are so much smaller than many established companies, and because they may
+grow quickly, a few other things are worth remembering when negotiating with startups:
+
+- **Cash versus equity**: If your risk tolerance is reasonably high, it’s reasonable to ask for
+  an offer with more equity and less cash.
+  Usually if a company is doing well, it is likely to “level up” lower salary levels later,
```

Comment:
meaning, if you accept a salary below market now, you can usually expect to have your salary raised as the company grows